packages/interchain_txs/src/helpers.rs

```diff
@@ -0,0 +1,47 @@
+use cosmos_sdk_proto::cosmos::base::abci::v1beta1::{MsgData, TxMsgData};
+use cosmwasm_std::{Binary, Deps, Reply, StdError, StdResult};
+use prost::{DecodeError, Message};
+
+use std::io::Cursor;
+
+pub fn parse_response(data: Binary) -> StdResult<Vec<MsgData>> {
```

Comment:
Do not forget to cover all public stuff (functions, structures, traits, etc) in our packages with [doc](https://doc.rust-lang.org/rust-by-example/meta/doc.html) comments, please.